server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/functions.js

```diff
@@ -637,14 +637,6 @@ function getSserversDetail(group) {
   return getJSONForTable(url, sessionDataVar);
 }
 
-/**
- * REST GET call for /manager,
- * stores it on a sessionStorage variable
- */
-function getManager() {
-  return getJSONForTable(REST_V2_PREFIX + '/manager', 'manager');
-}
-
```

Comment:
With this, there existed 2 getManager functions. This was the one being used since it's the last defined. The restv2 endpoint for manager was not compatible with the front end, leading to issues. Deleting this causes the rest/manager endpoint to be used instead for getManager, which is compatible with existing front end.